Jenkinsfile

```diff
@@ -56,15 +57,31 @@ def checkoutSrcs() {
 }
 
 def AMD64BuildCPU() {
-  def nodeReq = "ubuntu && amd64 && cpu-bare"
+  def nodeReq = "ubuntu && amd64 && cpu-build"
   def dockerTarget = "cpu_bare"
   def dockerArgs = ""
   node(nodeReq) {
     unstash name: 'srcs'
     echo "Building universal artifact for AMD64, CPU-only"
     sh """
     tests/ci_build/ci_build.sh ${dockerTarget} ${dockerArgs} tests/ci_build/build_via_cmake.sh
-    tests/ci_build/ci_build.sh ${dockerTarget} ${dockerArgs} tests/ci_build/create_wheel.sh
+    tests/ci_build/ci_build.sh ${dockerTarget} ${dockerArgs} tests/ci_build/create_wheel.sh manylinux1_x86_64
+    """
+    withAWS(credentials:'Neo-AI-CI-Fleet') {
+      s3Upload bucket: 'neo-ai-dlr-jenkins-artifacts', acl: 'Private', path: "${env.JOB_NAME}/${env.BUILD_ID}/artifacts/", includePathPattern:'python/dist/**'
+    }
+  }
+}
+
+def AMD64BuildGPU() {
+  def nodeReq = "ubuntu && amd64 && gpu-build"
+  node(nodeReq) {
+    unstash name: 'srcs'
+    echo "Building artifact for AMD64 with GPU capability. Using CUDA 8.0, CuDNN 7, TensorRT 4"
```

Comment:
Could you explain why Cuda 8?

Reply:
To maximize the range of compatible targets